### PR TITLE
Add recipe for uWSGI 2.0.17.1

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

**Comment by nsoranzo** [comment]
@natefoo I guess for building to work this needs your commit in https://github.com/galaxyproject/starforge-recipes/pull/4 plus some more fixes, I can rebase when it works.

**Comment by nsoranzo** [comment]
I made some progress here, e.g. found that the wheel directory name (and `name` inside `meta.yaml`) needs to match the PyPI one in a case-sensitive manner.

I am now stuck at strange uWSGI compilation issues: @natefoo can you help?

File: .travis.yml [hunk 1]
```diff
@@ -130,13 +130,22 @@ before_script:
...
-        $STARFORGE_CMD --debug test_wheel --wheels-config="$meta" $STARFORGE_IMAGE_ARGS "$wheel"
+        $STARFORGE_CMD --debug wheel --wheels-config="$meta" $STARFORGE_IMAGE_ARGS "$wheel"; STARFORGE_EXIT_CODE=$?
+        if [ "$STARFORGE_EXIT_CODE" -eq 0 ]; then
+            echo "Testing '$wheel' wheel"
+            $STARFORGE_CMD --debug test_wheel --wheels-config="$meta" $STARFORGE_IMAGE_ARGS "$wheel"
+        elif [ "$STARFORGE_EXIT_CODE" -eq 1 ]; then
+            echo "Building '$wheel' wheel failed"
+            BUILD_EXIT_CODE=1
+        else
+            echo "`starforge wheel` exited with code '$STARFORGE_EXIT_CODE', skipping wheel test"
```
Comment (natefoo) [inline .travis.yml]: Whoops, forgot to escape these backticks. But it's not failing the builds, so I'll let it finish running?

**Comment by nsoranzo** [comment]
@natefoo Is this normal https://travis-ci.org/galaxyproject/starforge-recipes/jobs/421708149#L801 ?

**Comment by natefoo** [comment]
That's due to the unescaped backticks I commented about.

**Comment by natefoo** [comment]
:+1: from me if you're good with my additions.

**Pull request merged**